core/controlplane/config/templates/stack-template.json

```diff
@@ -1593,6 +1593,14 @@
             "ToPort": 22
           },
           {{end -}}
+          {{ range $_, $etcdSubnet := .Etcd.Subnets -}}
+          {
+            "CidrIp": "{{ $etcdSubnet.InstanceCIDR }}",
+            "FromPort": 2379,
+            "IpProtocol": "tcp",
+            "ToPort": 2379
+          },
+          {{ end -}}
```

Comment:
This relaxation of the inbound rules to etcd clients allows ALL hosts within the same subnets as the etcd servers to connect to them.  In practice this should not be an issue because they are protected by TLS client/server certificates.  This is intended as a temporary step in order to faciliate the upgrade/migration

Reply:
Makes sense. Migrating to v0.11 results in a brand-new set of `SecurityGroupEtcd` + `SecurityGroupEtcd*Ingress`(that allows 2379 only within either the old or the new etcd cluster, but not between the two), and that's why we need this relaxation.